src/views/admin/rollen/RolleCreationView.vue

```diff
@@ -220,7 +220,11 @@
     });
 
     Object.values(RollenSystemRecht).forEach((enumValue: RollenSystemRecht) => {
-      if (enumValue !== RollenSystemRecht.MigrationDurchfuehren && enumValue !== RollenSystemRecht.CronDurchfuehren) {
+      if (
```

Comment:
Ich denke wir sollten in Utils eine Funktion bauen `hideSystemrecht(enumValue: RollenSystemRecht)`. Weniger Duplication. Mehr Sicherheit, dass es nicht auseinanderläuft.